src/functions/views/provider.php

```diff
@@ -133,5 +133,5 @@ function tribe_events_widgets_v2_is_enabled() {
 		return (bool) $env_var;
 	}
 
-	return ! defined( 'TRIBE_EVENTS_WIDGETS_V2_ENABLED' ) || TRIBE_EVENTS_WIDGETS_V2_ENABLED;
+	return defined( 'TRIBE_EVENTS_WIDGETS_V2_ENABLED' ) ? TRIBE_EVENTS_WIDGETS_V2_ENABLED : true;
```

Comment:
During the dev meeting we've set with `_DISABLED` constant names. 
Is this still using `_ENABLED` to keep back-compat?

Reply:
Nope - this hasn't changed per the dev meeting.

I'll change it.